SliderButton/tests/SliderButton-specs.js

```diff
@@ -0,0 +1,80 @@
+import {mount} from 'enzyme';
+import React from 'react';
+import SliderButton from '../SliderButton';
+
+const children = ['Light Speed', 'Ridiculous Speed', 'Ludicrous Speed'];
+
+describe('SliderButton Specs', () => {
+	test('should change value when clicking on the second button', () => {
```

Comment:
I think we can make it two different test cases.
One for emitting 'onChange' event and the other is related to set 'aria-valuetext'.
IMHO, having an assertion is good in a test method if possible. 
I don't need to prove onChange event is emitted but most of unit tests in enact have it. So, If you think it is necessary, please add it. 

```suggestion
	test('should emit an onChange event when changing value', () => {z
	const handleChange = jest.fn();
		const evt = {value: 1};
		const sliderButton = mount(
			<SliderButton onChange={handleChange}>
				{children}
			</SliderButton>
		);

		sliderButton.find('SliderButton').simulate('change', evt);

		const expected = 1;
		const actual = handleChange.mock.calls[0][0].value;
		expect(actual).toBe(expected);
    });

    test('should set "aria-valuetext" to value when value is changed', () => {
		const handleChange = jest.fn();
		const evt = {value: 1};
		const sliderButton = mount(
			<SliderButton onChange={handleChange}>
				{children}
			</SliderButton>
		);

		sliderButton.find('SliderButton').simulate('change', evt);

		const valueText = sliderButton.find('SliderButton').prop('aria-valuetext');
		expect(valueText).toBe('Ridiculous Speed');
	});
```